src/overlays/actors/ovl_Door_Killer/z_door_killer.h

```diff
@@ -4,11 +4,35 @@
 #include "ultra64.h"
 #include "global.h"
 
+/*
+ * Associated switch flag: (params >> 8) & 0x3F
+ * ((params >> 8) & 0x3F) == 0x3F means no switch flag is checked / set
+*/
+
+typedef struct{
+    /* 0x00 */ s16 objectId;
+    /* 0x04 */ Gfx* texture;
```

Comment:
A texture can't be of type `Gfx*` so there is something wrong here.
If it can be any pointer type, I suggest using `void*`.